docs/checkers/index.md

```diff
@@ -1118,6 +1118,29 @@ str                 sum                 super               tuple
 vars                zip
 ```
 
+(C9103)=
+
+### PEP8 name violation (C9103)
+
+This error is an extension of C0103 (invalid-name) which aims to provide a more detailed and beginner-friendly error message about how a variable name violated the PEP8 naming conventions.
```

Comment:
Don't reference C0103 here, just describe the checker as is. Then at the bottom of this entry you can add as a `*Note*` that this checker is intended to replace C0103.